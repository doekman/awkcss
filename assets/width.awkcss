BEGIN {
	width(20);
}

/^#/ {
	width(12);
}
