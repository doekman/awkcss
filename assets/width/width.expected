1234567890          
12345678901234567890
                    
#2345       
#234567890  
          doeke     
#234567890ab
